streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryTimeOrderedKeyValueBufferTest.java

```diff
@@ -29,4 +31,14 @@ public void bufferShouldAllowCacheEnablement() {
     public void bufferShouldAllowCacheDisablement() {
         new InMemoryTimeOrderedKeyValueBuffer.Builder<>(null, null, null).withCachingDisabled();
     }
+
+    @Test
+    public void bufferShouldAllowLoggingEnablement() {
+        new InMemoryTimeOrderedKeyValueBuffer.Builder<>(null, null, null).withLoggingEnabled(Collections.emptyMap());
```

Comment:
I think that if you were to actually pass in a map here and then assert you get the same map back from `logConfig()`, it would be enough test coverage.